content/en/docs/reference/command-line-tools-reference/feature-gates/ClusterTrustBundle.md

```diff
@@ -10,4 +10,13 @@ stages:
     defaultValue: false
     fromVersion: "1.27"
 ---
-Enable ClusterTrustBundle objects and kubelet integration.
+This feature gate exists in the Kubernetes API server and the controller manager.
+
+Used from the kube-apiserver, it enables ClusterTrustBundle support.
```

Comment:
Need the feature gate on the kubelet to use the volume right?

Reply:
Yes, that's documented elsewhere. This FG only controls the API and the signer.